docs/literate/src/files/getting_started_with_Trixi.jl

```diff
@@ -0,0 +1,287 @@
+#   Or you can open a file manager, find this directory, right-click inside and 
+#   choose `Open Terminal Here`.
+# - To install Julia execute the following commands in a Terminal:
+#   ````
+#   wget https://julialang-s3.julialang.org/bin/linux/x64/1.8/julia-1.8.5-linux-x86_64.tar.gz
+#   tar zxvf julia-1.8.5-linux-x86_64.tar.gz
+#   ````
+#   Now you can verify that Julia is installed entering `julia` command in a Terminal.
+
+# Then Julia will be invoked. To close Julia enter `exit()`.
+
+# ## Trixi installation
+
+# Trixi and its related tools are registered Julia packages. So installation of them is
+# running inside Julia. For appropriate work of Trixi you need to install 
+# [`Trixi`](https://github.com/trixi-framework/Trixi.jl),
+# [`Trixi2Vtk`](https://github.com/trixi-framework/Trixi2Vtk.jl),
+# [`OrdinaryDiffEq`](https://github.com/SciML/OrdinaryDiffEq.jl) and 
+# [`Plots`](https://github.com/JuliaPlots/Plots.jl).
+
+# - Open a Terminal.
+# - Invoke Julia executing `julia`.
+# - Execute following commands:
+#   ````
+#   import Pkg
+#   Pkg.add(["Trixi", "Trixi2Vtk", "OrdinaryDiffEq", "Plots"])
+#   ````
+
+# Now you have installed all these 
+# packages. [`Trixi2Vtk`](https://github.com/trixi-framework/Trixi2Vtk.jl) is a visualization
+# tool, [`OrdinaryDiffEq`](https://github.com/SciML/OrdinaryDiffEq.jl) provides time integration schemes
+# used by Trixi and [`Plots`](https://github.com/JuliaPlots/Plots.jl) can be used to directly
+# visualize Trixi results from the Julia REPL.
+
+# ## Usage
+
+# ### Files execution
+
+# Trixi has a big set
+# of [`examples`](https://github.com/trixi-framework/Trixi.jl/tree/main/examples), that can be taken
+# as basis for your future investigations.
+
+# Now execute one of them using 
+# [`trixi_include(...)`](https://trixi-framework.github.io/Trixi.jl/stable/reference-trixi/#Trixi.trixi_include-Tuple{Module,%20AbstractString})
+# function. `trixi_include(...)` expects
+# a single string argument with a path to a text file containing Julia code.
+# `joinpath(...)` join a path components into a full path. 
+# [`examples_dir()`](https://trixi-framework.github.io/Trixi.jl/stable/reference-trixi/#Trixi.examples_dir-Tuple{})
+# returns a path to the
+# [`examples`](https://github.com/trixi-framework/Trixi.jl/tree/main/examples) folder.
+
+# Let's execute a short two-dimensional problem setup. Which approximates the solution of
+# [`compressible Euler equations in 2D for an ideal gas`](https://trixi-framework.github.io/Trixi.jl/stable/reference-trixi/#Trixi.CompressibleEulerEquations2D)
+# with
+# [`weak blast wave initial condition`](https://trixi-framework.github.io/Trixi.jl/stable/reference-trixi/#Trixi.initial_condition_weak_blast_wave-Tuple{Any,%20Any,%20CompressibleEulerEquations2D})
+
+# Invoke Julia in a Terminal.
+# And execute following code. (*Remark:* you can ignore all arguments of trixi_include() except
+# path to the file) 
```

Comment:
Why _are_ these additional parts there, .e.g, the `module` and the callback stuff?

Reply:
'module' is here to enable using 'sol' variable for 'plot' function, otherwise error appears, that 'sol' is undefined.
'callbacks' are used to prevent from saving the solution, problem is in [this part of hohqmesh_tutorial](https://trixi-framework.github.io/Trixi.jl/stable/tutorials/hohqmesh_tutorial/#Running-and-visualizing-an-unstructured-simulation). Here converting actions on .h5 files are performed, and error because of inconsistent of mesh files appears. ('trixi_include()' is used only in hohqmesh_tutorial and in new one, so there was no problem before)

Reply:
Is it possible to clean the `out` directory after this tutorial is run? this should fix the issue with the HOHQMesh tutorial. Alternatively, we can try to clean the `out` directory before the HOHQMesh tutorial is run.

Reply:
I applied first option. It works, thank you!